pom.xml

```diff
@@ -159,6 +168,18 @@
         </plugins>
     </build>
 
+<dependencyManagement>
+  <dependencies>
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>libraries-bom</artifactId>
+      <version>7.0.0</version>
+      <type>pom</type>
+      <scope>import</scope>
+    </dependency>
+  </dependencies>
+</dependencyManagement>
+
```

Comment:
Same here. This can be moved to the plugin

Reply:
Fixed in e3988a60.